src/lib/pickles/pickles.ml

```diff
@@ -2867,8 +2875,10 @@ module Make_str (_ : Wire_types.Concrete) = struct
                               let vk = As_prover.Ref.get vk in
                               Side_loaded.in_prover side_loaded_tag vk ) ;
                           let vk =
-                            exists Side_loaded_verification_key.typ
-                              ~compute:(fun () -> As_prover.Ref.get vk)
+                            exists
+                              (Side_loaded_verification_key.typ
+                                 Plonk_types.Features.none ) ~compute:(fun () ->
```

Comment:
It's a dummy value that's, in the end, used for decoding `Maybe` values. When the flag is false, no value has been encoded, so that one must have a placeholder. This is this `none` value. Since the encoded value is polymorphic, the dummy value must resurface where we keep the polymorphism - like here. 

However, the `none` value is also consistent with the fact that the feature flags are `none_bool`.